google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/StreamWriter.java

```diff
@@ -206,7 +205,9 @@ private StreamWriter(Builder builder) throws IOException {
             "Trying to enable connection pool in non-default stream.");
       }
 
-      BigQueryWriteClient client = getBigQueryWriteClient(builder);
+      // We need a client to perform some getWriteStream calls.
+      BigQueryWriteClient client =
```

Comment:
why do we need to create another client here if the passed in client is null? 

we won't use the created client